react-frontend/src/sagas/loginSaga.test.js

```diff
@@ -0,0 +1,89 @@
+import { put, takeEvery, call } from "redux-saga/effects";
+import { loginApi, userLogin } from "sagas/loginSaga";
+import PostJson from "utils/postJson";
+import actionGenerator from "utils/actionGenerator";
+import SuccessResponse from "../../../mock-responses/login/success_response.json";
+import UnauthorizedResponse from "../../../mock-responses/login/unauthorized_response.json";
+import InternalServerError from "../../../mock-responses/login/internal_server_error.json";
```

Comment:
DO NOT use relative paths like this. Please stick to using absolute paths, like we are using for other imports.